pages/common/mktemp.md

```diff
@@ -1,16 +1,16 @@
 # mktemp
 
 > Create a temporary file or directory.
-> More information: <https://www.gnu.org/software/autogen/mktemp.html>.
+> More information: <https://manned.org/mktemp.1>.
 
-- Create an empty temporary file and return the absolute path to it:
+- Create an empty temporary file and print the absolute path to it:
 
 `mktemp`
 
-- Create a temporary directory and return the absolute path to it:
+- Create an empty temporary file with a given suffix and print the absolute path to file:
 
-`mktemp -d`
+`mktemp --suffix "{{.ext}}"`
 
-- Create a temporary file with a specified suffix:
+- Create a temporary directory and print the absolute path to it:
 
-`mktemp --suffix "{{.txt}}"`
+`mktemp --directory`
```

Comment:
I suggest not using long options that are visibly not supported on `macOS` (see `*`):
```suggestion
`mktemp -d`
```

\* : 
```sh
$ echo $OSTYPE
darwin20.0
$ mktemp --directory
mktemp: illegal option -- -
usage: mktemp [-d] [-q] [-t prefix] [-u] template ...
       mktemp [-d] [-q] [-u] -t prefix
```

Reply:
Are you using GNU Coreutils?

Reply:
> Are you using [GNU Coreutils mktemp](https://www.gnu.org/software/coreutils/manual/coreutils.html#mktemp-invocation)?

No, I am using BSD's `mktemp` (which is the default one on macOS, I guess).

> I think we should prefer by default some specific set of utilities GNU/BSD or anything else not to mix them in an unpredictable way.

Not sure to understand. :innocent:
But other maintainers will probably have an opinion about it. :thought_balloon:

Reply:
Indeed, we do prefer long arguments where possible. However, in this instance they are not portable - so we should use the short ones which do seem to be portable.

Reply:
What to prefer:

- `(non-portable long option: --directory)` in command description as now
- `mktemp -d|--directory` supposing that all portable options are going first

?